Initializes the December Newsletter (#230)
readthedocs/ethicalads.io: content/posts/newsletter-december-2021.md

Title: EthicalAds Newsletter - December 2021
Date: Jan 7, 2022
description: Updates from December, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community
authors: Ra Cohen
image: /images/posts/2021-december-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@aaronburden?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Aaron Burden</a> on <a href="https://unsplash.com/s/photos/winter?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## New features

> Happy New Years! Here are the major features we released in the last month:

Reviewer: ```suggestion
Happy New Year! Here are the major features we released in the last month:
```

I think this is the more common phrasing?

* Rolled out our new [floating ad](https://ethical-ad-client.readthedocs.io/en/latest/#placement-style) type for all of the projects on Read the Docs.
* Debuted the beta for our [sponsorship platform](https://www.ethicalads.io/sponsorship-platform/).

You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## December advertising stats

In December 2021:

* We generated **$31,782.17** for our publishers.
* We had **102 publishers** on our network with at least one paid ad impression.
* We had **13,654,617** paid ad views across the world.

> As expected, our views did decrease precipitously during the Holidays,

Reviewer: ```suggestion
As expected, our views did decrease precipitously during the holidays,
```

but our revenue was actually still larger than September's and fairly close to October's.

> Imagine our surprise to see that our revenue actually increased in November by just over 8%!

Reviewer: I don't quite understand what this is saying?

Author: Yeah It's confusing taking a look for a second time. I'll take another crack at it.

## Tricky bits

Onboarding new publishers remains our largest hurdle to growth.
We're attempting to combat this by creating and sharing interesting new content,

> like our first [publisher interview](link) with the creator of daily.dev Ido Shamun.

Reviewer: #TODO 😆

## Upcoming features

The major features in our upcoming roadmap:

> * Improved keyword detection in the ad client.

Reviewer: These are the same features as last month, aren't they? Seems we should probably mention that, or change them?

Author: I like mentioning it since, I think they are the same as last month.

Title: EthicalAds Newsletter - December 2021

> Date: Jan 7, 2022

Reviewer 2: ```suggestion
Date: Jan 14, 2022
```

description: Updates from December, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community
authors: Ra Cohen
image: /images/posts/2021-december-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@aaronburden?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Aaron Burden</a> on <a href="https://unsplash.com/s/photos/winter?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

Happy New Year! December was a shorter month for us at EthicalAds so this update will be short and sweet. 

## New features

These are the major features we released in the last month:

> * Rolled out our new [floating ad](https://ethical-ad-client.readthedocs.io/en/latest/#placement-style) type for all of the projects on Read the Docs.

Reviewer 2: ```suggestion
```

* Debuted the beta for our [sponsorship platform](https://www.ethicalads.io/sponsorship-platform/).
* Released our [advertising prospectus](https//www.ethicalads.io/prospectus/ethicalads-advertiser-prospectus.pdf) for 2022.

You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## December advertising stats

In December 2021:

* We generated **$31,782.17** for our publishers.
* We had **102 publishers** on our network with at least one paid ad impression.
* We had **13,654,617** paid ad views across the world.

As expected, our views did decrease precipitously during the holidays (down around 40% from November),
but our revenue only decreased by 8% over the same time.

> This was due in part to many pre-orders for January, 

Reviewer 2: ```suggestion
This was due mostly to some large, inexpensive campaigns wrapping up
```
